test/unit/middlewares/requests.test.ts

```diff
@@ -31,9 +31,7 @@ describe("Create Request Validators", function () {
     it("should pass validation for a valid create request", async function () {
       req = {
         body: validOooStatusRequests,
-        query: {
-          dev: "true",
-        },
+        query: {},
```

Comment:
Can we remove this query also as passing this as empty

Reply:
Done